lib/webtask.js

```diff
@@ -14,7 +14,9 @@ var getFrom = require('lodash.get');
  *
  * @constructor
  */
-function Webtask (sandbox, token, meta) {
+function Webtask (sandbox, token, options) {
```

Comment:
It should not be; the `Webtask` constructor is designed only for internal consumption. It is exposed for `typeof` checks alone.